### PR TITLE
R markdown report

### PR DESCRIPTION
**Pull request by mult1fractal**
* full markdown report
* working on large test profile

* stub crashes somewhere (will investigate when manuscript is submitted)
* markdown report needs some fine-tuning (make it more beautiful)

### CODE REVIEW

**Comment by replikation** [comment]
@mult1fractal dry run needs some adjustments `cp: cannot stat '*.input': No such file or directory` might be something missing in the stub

File: phage.nf [hunk 1]
```diff
@@ -193,6 +195,10 @@ workflow {
 
         prepare_results_wf(results, prediction_channel)
 
+        // markdown report input
```
Comment (replikation) [inline phage.nf:198]: remove some of the old comments that are not needed

File: phage.nf [hunk 2]
```diff
@@ -206,14 +212,49 @@ workflow {
         phage_annotation_wf(annotation_channel)
         checkV_wf(annotation_channel)
         phage_tax_classification_wf(annotation_channel)
+
+        // markdown report input
+        //annotation_markdown = phage_annotation_wf.out.concat(checkV_wf.out)//.view()
```
Comment (replikation) [inline phage.nf:217]: remove some of the old comments that are not needed

File: phage.nf [hunk 3]
```diff
@@ -193,6 +195,10 @@ workflow {
 
         prepare_results_wf(results, prediction_channel)
 
+        // markdown report input
+        //identify_markdown = prepare_results_wf.out//.view()
+        //upsetr_markdown = prepare_results_wf.out
```
Comment (replikation) [inline phage.nf:200]: remove some of the old comments that are not needed

**Comment by mult1fractal** [comment]
in the git action it action it fails during mrkdown report sumary

File: phage.nf (hunk 3 above)
Comment (replikation) [reply phage.nf:200]: also line 199

File: phage.nf [hunk 4]
```diff
@@ -206,14 +212,49 @@ workflow {
...
         phage_tax_classification_wf(annotation_channel)
+
+        // markdown report input
+        //annotation_markdown = phage_annotation_wf.out.concat(checkV_wf.out)//.view()
+        phage_annotation_wf.out
+
+
+
+        // [all_pos_phage, large, /tmp/nextflow-phages-mike/7f/8f2fc9e2d84a96b149f8df56d9b85a/large/chromosomefile.tbl, /tmp/nextflow-phages-mike/7f/8f2fc9e2d84a96b149f8df56d9b85a/large/annotationfile.tbl, all_pos_phage, /tmp/nextflow-phages-mike/92/22d3a7d3cada17ec8fab7470800720/all_pos_phage_quality_summary.tsv]
+        //[all_pos_phage, small, /tmp/nextflow-phages-mike/7f/8f2fc9e2d84a96b149f8df56d9b85a/small/chromosomefile.tbl, /tmp/nextflow-phages-mike/7f/8f2fc9e2d84a96b149f8df56d9b85a/small/annotationfile.tbl, all_pos_phage, /tmp/nextflow-phages-mike/92/22d3a7d3cada17ec8fab7470800720/all_pos_phage_quality_summary.tsv]
```
Comment (replikation) [inline phage.nf:223]: remove some of the old comments that are not needed

File: workflows/process/prepare_results/seqkit_tool_agreements.nf [hunk 5]
```diff
@@ -3,7 +3,8 @@ process seqkit_tool_agreements {
 /*         errorStrategy 'ignore' */
         label 'seqkit'
     input:
-        tuple val(name), path(toolagreement_per_contig), path(fasta)
+        tuple val(name), path(toolagreement_per_contig)
+        tuple val(name), path(fasta)
```
Comment (replikation) [inline workflows/process/prepare_results/seqkit_tool_agreements.nf:7]: how does this work.? what happens when multiple samples are used? nextflow is not joining them by correct name?

**Review by replikation — CHANGES_REQUESTED** [review]
see line comments

**Pull request merged**